cmd/sample-service/main.go

```diff
@@ -152,6 +155,10 @@ func (w *x509Watcher) OnX509ContextUpdate(c *workloadapi.X509Context) {
 		return
 	}
 
+	// Set max connection lifetime to be slightly more than the frequency of SVID updates from SPIRE (2m vs 1m)
+	// so that we do don't close connection before establishing a new one
```

Comment:
```suggestion
	// so that we don't close connection before establishing a new one
```

Reply:
I'll send a follow-up PR for this so that we can get the core changes of this PR in main. There's another PR I'm planning to send out that adjusts TTL configs on `main` which will also need to adjust this.